Content.Client/UserInterface/HumanoidProfileEditor.cs

```diff
@@ -967,5 +1012,56 @@ private void OnCheckBoxToggled(BaseButton.ButtonToggledEventArgs args)
                 PreferenceChanged?.Invoke(Preference);
             }
         }
+
+        private class CrewUniformPreferenceSelector : Control
+        {
+            private readonly RadioOptions<int> _optionButton;
+
+            public event Action<CrewUniformPreference>? CrewUniformPreferenceChanged;
+
+            public CrewUniformPreference UniformPreference
+            {
+                get => (CrewUniformPreference) _optionButton.SelectedValue;
+                set => _optionButton.SelectByValue((int) value);
+            }
+
+            public CrewUniformPreferenceSelector()
+            {
+                _optionButton = new RadioOptions<int>(RadioOptionsLayout.Horizontal)
+                {
+                    FirstButtonStyle = StyleBase.ButtonOpenRight,
+                    ButtonStyle = StyleBase.ButtonOpenBoth,
+                    LastButtonStyle = StyleBase.ButtonOpenLeft
+                };
+
+                _optionButton.AddItem(Loc.GetString("Default"), (int) CrewUniformPreference.Default);
+                _optionButton.AddItem(Loc.GetString("Trek"), (int) CrewUniformPreference.Trek);
+                _optionButton.AddItem(Loc.GetString("Next Gen"), (int) CrewUniformPreference.NextGen);
+                _optionButton.AddItem(Loc.GetString("Enterprise"), (int) CrewUniformPreference.Enterprise);
+                _optionButton.AddItem(Loc.GetString("Federation"), (int) CrewUniformPreference.Federation);
```

Comment:
Can't you just use the RadioOptions directly? Also this shouldn't have the enum hardcoded into it.